stubs/console-menu/consolemenu/prompt_utils.pyi

```diff
@@ -0,0 +1,46 @@
+from typing import Any, NamedTuple
+
+from consolemenu.screen import Screen
+from consolemenu.validators.base import BaseValidator as BaseValidator, InvalidValidator as InvalidValidator
+
+class InputResult(NamedTuple):
+    input_string: str
+    validation_result: bool
+
+class PromptFormatter:
+    @staticmethod
+    def format_prompt(
+        prompt: str | None = ...,
+        default: str | None = ...,
+        enable_quit: bool = ...,
+        quit_string: str = ...,
+        quit_message: str = ...,
+    ) -> str: ...
+
+class PromptUtils:
+    def __init__(self, screen: Screen, prompt_formatter: PromptFormatter | None = ...) -> None: ...
+    @property
+    def screen(self) -> Screen: ...
+    def clear(self) -> None: ...
+    def confirm_answer(self, answer: str, message: str | None = ...) -> bool: ...
+    def enter_to_continue(self, message: str | None = ...) -> None: ...
+    def input(
+        self,
+        prompt: str | None = ...,
+        default: str | None = ...,
+        validators: list[BaseValidator] | None = ...,
```

Comment:
Does it really have to be a list? If possible we should type this as Sequence or even Iterable.